### DISCUSSION TITLE
HC_envSet/Docker_On_Linux/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#52] [Show and tell]
# HC_envSet/Docker_On_Linux/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://ctf.probius.xyz/HC_envSet/Docker_On_Linux/

**Comment by GitZhiQing** [comment]
# 啊，又打开了新思路！

**Comment by GitZhiQing** [comment]
所有功能均正常，但是尝试在vscode的REGISTRIES连接dockerhub管理自己的镜像时，被拒绝连接。搜索了很久这个问题，似乎是一些代理及https相关的问题，但仍然不能找到相关问题的解决方式，请问有这个问题的解决参考方案嘛呜呜呜。
报错信息：request to https://hub.docker.com/v2/users/login failed, reason: connect ECONNREFUSED 31.13.85.169:443
![1.jpg](https://cdnjson.com/images/2023/10/30/1.jpg)

**Reply by GitZhiQing** [reply]
该问题的截图
https://cdnjson.com/images/2023/10/30/1.jpg

**Reply by ProbiusOfficial** [reply]
因为dockerhub已经被GFW拦截，国内已经无法直接访问了。

**Comment by Xia0-0** [comment]
作为一个小白，这里着实不太会但无妨，没说一定要按顺序食用呀（就是浪^O^）
